salt/metalk8s/_internal/python-kubernetes/init.sls

```diff
@@ -0,0 +1,11 @@
+#
+# Python kubernetes requirements
+#
+# Available states
+# ================
+#
+# * installed   -> Install python kubernetes library
```

Comment:
Details, here it says  `python kubernetes library` but in the state it's written `Python Kubernetes client`.

Reply:
Or sir will changes the case but what do you prefer `client` or `library` ? Or maybe `client library` :)

Reply:
I don't really care, just to say it's not consistent. :p